src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl

```diff
@@ -97,7 +97,7 @@ end
     return z <= h
 end
 
-@inline bottom(i, j, k, ibg::GFIBG) = @inbounds ibg.immersed_boundary.bottom_height[i, j]
+@inline z_bottom(i, j, k, ibg::GFIBG) = @inbounds ibg.immersed_boundary.bottom_height[i, j]
```

Comment:
is anywhere `z_bottom` assumed to be on the face interface of the grid?

Reply:
hmm I think we should define it that way. This function is actually not quite right; it should return the true numerical height rather than the bathymetric height